src/producer.ts

```diff
@@ -24,6 +24,7 @@ export class Producer {
       options.sqs ||
       new SQSClient({
         ...options,
+        useQueueUrlAsEndpoint: options.useQueueUrlAsEndpoint ?? true,
```

Comment:
Just need to check that `true` is also the default for AWS SDK.

Reply:
It is: https://github.com/aws/aws-sdk-js-v3/blob/9083ece20cef1748cb86beae96e351e1bf2c00b8/packages/middleware-sdk-sqs/src/queue-url.ts#L41